neqo-bin/src/bin/client/http09.rs

```diff
@@ -253,15 +253,21 @@ impl<'b> Handler<'b> {
                 return Ok(());
             }
             Some(maybe_out_file) => {
+                let mut buf_writer = maybe_out_file
+                    .take()
+                    .map(|file| BufWriter::with_capacity(BUFWRITER_BUFFER_SIZE, file));
```

Comment:
Why construct a new `BufWriter` on each `read` call? Shouldn't the `BufWriter` be wrapping the `File` for the lifetime of the `File`? That would be consistent with the `http3.rs` implementation.